debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java

```diff
@@ -354,6 +354,9 @@ else if (eventHeader instanceof EventHeaderV4) {
             // Capture that we've completed another event ...
             offsetContext.completeEvent();
 
+            // Set the value of lastOffset
```

Comment:
```suggestion
            // update last offset used for logging
```

Reply:
It's borderline useless to have a comment here anyways, but "set the value" is not adding any value. So I've reworded it a bit to point out the purpose.